rllib/connectors/common/add_states_from_episodes_to_batch.py

```diff
@@ -0,0 +1,286 @@
+import math
```

Comment:
Result of disseminating all default connector pieces (env-to-module, module-to-env, learner). Some of these individual pieces are now shared (e.g. both env-to-module AND learner will use the `AddObservationsFromEpisodesToBatch`).